test/Telegram.Bot.Tests.Integ/Sending Messages/SendingPhotoMessageTests.cs

```diff
@@ -116,15 +117,15 @@ public SendingPhotoMessageTests(TestsFixture fixture, EntityFixture<Message> cla
         {
             await _fixture.SendTestCaseNotificationAsync(FactTitles.ShouldParseMessageCaptionEntitiesIntoValues);
 
-            var entityValueMappings = new (MessageEntityType Type, string EntityBody, string EncodedEntity)[]
+            (MessageEntityType Type, string EntityBody, string EncodedEntity)[] entityValueMappings = new (MessageEntityType Type, string EntityBody, string EncodedEntity)[]
```

Comment:
I regret writing tests like this. No readable much.
We are now trying to have less variables in test methods and use more literals.

Here is an example:
<https://github.com/TelegramBots/Telegram.Bot/blob/63529448cec1a497808c859fcdadd483e64690d4/test/Telegram.Bot.Tests.Integ/Sending%20Messages/AlbumMessageTests.cs#L208-L212>